src/openforms/conf/base.py

```diff
@@ -859,6 +859,8 @@
 DIGID_METADATA = config("DIGID_METADATA", "")
 SSL_CERTIFICATE_PATH = config("SSL_CERTIFICATE_PATH", "")
 SSL_KEY_PATH = config("SSL_KEY_PATH", "")
```

Comment:
These are no longer used anywhere I think, so it might be good to remove them

Reply:
check, is done.